src/Text/Pandoc/Readers/JATS.hs

```diff
@@ -303,15 +304,22 @@ parseBlock (Elem e) =
                                                 Just ws' -> let tot = sum ws'
                                                             in  ColWidth . (/ tot) <$> ws'
                                                 Nothing  -> replicate numrows ColWidthDefault
-                      let toRow = Row nullAttr . map simpleCell
+
+                      -- How do I parse an Element into Blocks?
+                      -- parseCell takes an element and returns a StateT JATSState Blocks, but I need it to return the unwrapped value
+                      let parseCell = parseMixed plain . elContent
+
+                      let elementToCell element = cell (toAlignment element) (RowSpan $ toRowSpan element) (ColSpan $ toColSpan element) mempty
```

Comment:
Here is where I am having trouble. Right now, I'm creating an empty `Cell`, but I really want to be able to use `parseCell` to parse the content of the xml cell into the Pandoc cell

Reply:
All parsing happens ins the `StateT JATSState m` type, a.k.a. `JATS m`. It is an instance of `Monad`, so we can use `cellContent <- parseCell element` while we are in a `do` block, as is the case here.
I could probably free an hour tomorrow if you want to pair on this.

Reply:
Thanks for the quick response @tarleb! Running `cellContent <- parseCell element` makes sense, but I think what's tripping me up is that I would want to run it in the context of `elementToCell`, which is not in a do block. 

I'd love to do a quick pair! I'm normally available between 8-5 CST (GMT -6), but also wouldn't mind stretching on either end. Let me know when works for you!

Reply:
Would [14:30 UTC](https://time.is/1430_28_Mar_2023_in_UTC) work for you? If so, please send me an invite via mail.